scripts/tiling/tiling.py

```diff
@@ -0,0 +1,72 @@
+""" Script for creating an FOV list json file that contains a grid of FOVs
+generated from a single FOV json.
+"""
+
+import os
+import sys
+import json
+import datetime
+import argparse
+import copy
+import numpy as np
+
+def tile(fov_list_json_file, xn, yn, overlap):
+    ''' Using a template json file, creates another fov json that includes
+    the tiled version of the original FOV.
+    Args:
+        fov_list_json_file: The FOV json containing the FOV to be tiled. The
+            resulting tiled json is created in the same directory as this file.
+        xn: The number of tiles in the x direction.
+        yn: The number of tiles in the y direction.
+        overlap: The degree of overlap between tiles. Must be between 0-1.
```

Comment:
A use case for some could be to leave spacing between adjacent FOVs instead of overlapping. Additionally, it would be useful to be able to specify different overlap/spacing in each the x and y dimensions.

Reply:
Done